src/app/QuickStarts/quickstarts/automated-rules-quickstart.tsx

```diff
@@ -78,86 +75,74 @@ In this quick start, you will use [APP] to create an automated rule that will st
 
     `,
     tasks: [
-      {
-        title: 'Go to the Automated Rules page',
-        description: `1. Click the [Automated Rules]{{highlight nav-automatedrules-tab}} tab in the [APP] console navigation bar.`,
-        review: {
-          instructions: '#### Verify that you see the Automated Rules page.',
-          failedTaskHelp:
-            'If you do not see the navigation bar, you can click the `☰` button in the [top left corner of the page]{{highlight nav-toggle-btn}}.',
-        },
-      },
       {
         title: 'Create a new Automated Rule',
         description: `
-1. Click the [Create]{{highlight create-rule-btn}} button.
-[Read the [About Automated Rules]{{highlight about-rules}} section of the for more information.]{{admonition tip}}
-`,
+1. In the [APP] console navigation bar, click [Automated Rules]{{highlight nav-automatedrules-tab}}.
+2. Click [Create]{{highlight create-rule-btn}}.
+        `,
         review: {
           instructions: '#### Verify that you see the Automated Rules creation form.',
-          failedTaskHelp: 'If you do not see the creation form, follow the previous steps again.',
+          failedTaskHelp:
+            'If you do not see the navigation bar, click the [menu button]{{highlight nav-toggle-btn}} on the masthead.',
         },
       },
       {
         title: 'Fill out the Automated Rule form',
         description: `
-The Automated Rule creation form has several fields that you can fill out to create a new rule. Each field has helper text that explains what the field is for.
-
-**The most important field is the [Match Expression]{{highlight rule-matchexpr}} field.** This field is used to match one or more target JVMs. The match expression is a Java-like code snippet that is matched against each target JVM. For example, if you wanted to match all discovered target JVMs, try using the match expression: \`true\`{{copy}}.
+To create a new rule, you'll use the Automated Rule creation form which has multiple fields with helpful text to guide you. Each field is labeled to explain its purpose and help you fill out the form correctly.
 
-[Use the [Match Expression Visualizer]{{highlight match-expr-card}} to test your match expression against the target JVMs that are currently discovered by [APP]. Matched targets will be unfaded in the Graph view, and listed in the List View.]{{admonition tip}}
+**The most important field is the [Match Expression]{{highlight rule-matchexpr}} field.** This field is used to match one or more target JVMs. The match expression is a Java-like code snippet that is matched against each target JVM.
```

Comment:
The first paragraph might not be needed as it is not adding any more additional info/clarifty to the UI. 

How about, here, we just explain what a match expression is and its example?